### PR TITLE
My patch for cert expiry time validation

### PR DESCRIPTION
**Pull request by hussamnasir**

### CODE REVIEW

File: tools/cert_utils.py [hunk 1]
```diff
@@ -161,9 +161,17 @@ def make_cert(uuid, email, urn, signer_cert_file, signer_key_file, csr_file,
     config = pm.getService('config')
     ssl_config_file = config.get("chapi.ssl_config_file")
 
-    # Check validity of args (weak, I know)
+    # Cert Expiry time cannot be greater than signer expiry. 
+    cert_lifetime=365
```
Comment (nbastin) [inline tools/cert_utils.py]: I'm not sure why this is here - why don't we just use `days`?  This effectively makes the `days` argument meaningless.

Comment (hussamnasir) [reply tools/cert_utils.py]: i agree, but the problem is days variable was conflicting the "datetime.days" function.

Comment (nbastin) [reply tools/cert_utils.py]: I'm not sure I understand - it should not do such a thing.  I presume you mean in `datetime.timedelta`?  You are free to do `datetime.timedelta(days=days)`, as that will work properly.  The problem here is you have effectively hard coded the days value, so any caller that passes a value different from 365 will not get the cert they want.

Comment (hussamnasir) [reply tools/cert_utils.py]: okay i get what you are saying now. just change 
cert_lifetime = days
is that right ?

Comment (nbastin) [reply tools/cert_utils.py]: I would just remove this line and use `days` everywhere you were previously using `cert_lifetime` - there's no need for an extra variable.

Comment (hussamnasir) [reply tools/cert_utils.py]: I wasnt sure if the context for the variable would be distored by 
"cert_lifetime = remaining.days - 1" where days in this case is a datetime function

Comment (nbastin) [reply tools/cert_utils.py]: Their scopes are unambiguous, it won't cause any problems.

Comment (hussamnasir) [reply tools/cert_utils.py]: agreed. Will close this and create a new pull request

**Pull request closed without merging**